### PR TITLE
New admin check for permissions

### PR DESCRIPTION
**Pull request by rememberlenny**

### CODE REVIEW

**Review by msecret — CHANGES_REQUESTED** [review]
I think we should add tests for the new user actions, the new case statement in the user store and the new method in the user store.

**Comment by rememberlenny** [comment]
Added test for `new user actions`.

**Comment by rememberlenny** [comment]
Added `case` conditions for `CURRENT_UAA_INFO_FETCH`.

**Comment by rememberlenny** [comment]
Added the `isAdmin()` related checks.

**Comment by rememberlenny** [comment]
Finished up the tests around the points mentioned. Good to go! @msecret

File: static_src/test/unit/stores/user_store.spec.js [hunk 1]
```diff
@@ -433,6 +433,73 @@ describe('UserStore', function () {
     });
   });
 
+  describe('on CURRENT_UAA_INFO_RECEIVED', function () {
+    it('should not fail when groups is missing in uaaInfo', function () {
+      const currentUaaInfo = { guid: '1234' };
```
Comment (msecret) [inline static_src/test/unit/stores/user_store.spec.js]: These tests in each `it` block seem to have repeated information. Is it possible to move the data to a `beforeEach` like you did below?

**Review by msecret — APPROVED** [review]
One comment about making tests less repeated.

**Pull request merged**